src/components/views/messages/ReactionsRowButton.tsx

```diff
@@ -100,27 +105,58 @@ export default class ReactionsRowButton extends React.PureComponent<IProps, ISta
                     content={content}
                     reactionEvents={reactionEvents}
                     visible={this.state.tooltipVisible}
+                    customReactionImagesEnabled={this.props.customReactionImagesEnabled}
                 />
             );
         }
 
         const room = this.context.getRoom(mxEvent.getRoomId());
         let label: string | undefined;
+        let customReactionName: Optional<string>;
```

Comment:
I think this is not our preferred way of doing an optional var - I think `string | undefined` is often preffered

Reply:
The reason I did this is because the `findIn` function returns `Optional<string>`